pkgs/applications/networking/instant-messengers/seren/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchurl
+, alsaLib
+, libopus
+, libogg
+, gmp
+, ncurses
+}:
+
+stdenv.mkDerivation rec {
+  pname = "seren";
+  version = "0.0.21";
+
+  buildInputs = [ alsaLib libopus libogg gmp ncurses ];
+
+  src = fetchurl {
+    url = "http://holdenc.altervista.org/seren/downloads/${pname}-${version}.tar.gz";
+    sha256 = "sha256-adI365McrJkvTexvnWjMzpHcJkLY3S/uWfE8u4yuqho=";
+  };
+
+  meta = with lib; {
+    description = "A simple ncurses VoIP program based on the Opus codec";
+    longDescription = ''
+      Seren is a simple VoIP program based on the Opus codec
+      that allows you to create a voice conference from the terminal, with up to 10
+      participants, without having to register accounts, exchange emails, or add
+      people to contact lists. All you need to join an existing conference is the
+      host name or IP address of one of the participants.
+    '';
+    homepage = "http://holdenc.altervista.org/seren/";
+    changelog = "http://holdenc.altervista.org/seren/";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ matthewcroughan nixinator ];
+    platforms = [
+      "x86_64-linux" "i686-linux"
+      "aarch64-linux" "armv7l-linux" "armv7a-linux" "armv6l-linux" "armv5tel-linux"
+      "riscv64-linux"
+      "powerpc64-linux"
+    ];
+  };
+}
+
```

Comment:
```suggestion
    platforms = platforms.linux;
  };
}
```

Reply:
platforms.linux would be less accurate though wouldn't it? I confirmed all the packages in my list, it does not compile for the rest, such as riscv32-linux.

Reply:
I doubt that most parts of riscv32-linux compile. Platforms right now is mostly about x86_64, aarch64 linux and mac. You can find more information about how well we support each platform here https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md. risc is not even on that list and would probably be tier 4 or lower.